libs/sdk-ui-dashboard/api/sdk-ui-dashboard.api.md

```diff
@@ -12,9 +12,10 @@ import { AnyAction } from '@reduxjs/toolkit';
 import { CaseReducer } from '@reduxjs/toolkit';
 import { CaseReducerActions } from '@reduxjs/toolkit';
 import { ComponentType } from 'react';
-import { DashboardDateFilterConfigMode } from '@gooddata/sdk-backend-spi';
+import { DashboardDateFilterConfigMode } from '@gooddata/sdk-model';
 import { DataViewFacade } from '@gooddata/sdk-ui';
 import { DateFilterGranularity } from '@gooddata/sdk-backend-spi';
```

Comment:
Seems there are some forgotten import from the SPI - see also the other _2 new imports in this file.